package.json

```diff
@@ -29,7 +29,7 @@
     "tape": "^4.0.0",
     "testling": "^1.7.1",
     "travis-multirunner": "^3.0.0",
-    "webrtc-adapter": "^3.4.3"
+    "webrtc-adapter": "^6.0.2"
```

Comment:
We've moved a bunch of otalk repos to have adapter as a peer dependency, and I guess didn't get this one yet? Let's do that. @legastero @fippo objections?

Reply:
Ignore me, we already did that here, and just moved it from dep to devDep https://github.com/otalk/RTCPeerConnection/commit/598f7dd3f8bcebab665fff990bbff01570923847#diff-b9cfc7f2cdf78a7f4b91a753d10865a2